pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -189,19 +189,41 @@ func TargetCluster(targetCluster string) MurModifier {
 	}
 }
 
-func Account(cluster string, tier toolchainv1alpha1.NSTemplateTier) MurModifier {
+type AccountOption func(ua *toolchainv1alpha1.UserAccountEmbedded)
 
+func SyncIndex(index string) AccountOption {
+	return func(ua *toolchainv1alpha1.UserAccountEmbedded) {
+		ua.SyncIndex = index
+	}
+}
+
+// Account sets the first account on the MasterUserRecord
+func Account(cluster string, tier toolchainv1alpha1.NSTemplateTier, options ...AccountOption) MurModifier {
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
+		mur.Spec.UserAccounts = []toolchainv1alpha1.UserAccountEmbedded{}
+		return AdditionalAccount(cluster, tier, options...)(mur)
+	}
+}
+
+// AdditionalAccount sets an additional account on the MasterUserRecord
+func AdditionalAccount(cluster string, tier toolchainv1alpha1.NSTemplateTier, options ...AccountOption) MurModifier {
 	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
-		// set the user account
 		templates := nstemplateSetFromTier(tier)
-		mur.Spec.UserAccounts = append(mur.Spec.UserAccounts, toolchainv1alpha1.UserAccountEmbedded{
-			TargetCluster: "member-cluster",
+		ua := toolchainv1alpha1.UserAccountEmbedded{
+			TargetCluster: cluster,
+			SyncIndex:     "1", // default value
```

Comment:
Could you please align the values? 
https://github.com/codeready-toolchain/toolchain-common/blob/3a94d31a9796c5c5fb2efe5f42bfaf0618924d0f/pkg/test/masteruserrecord/master_user_record.go#L137

Reply:
or, if you slightly change the `newEmbeddedUa` function (to take the given NSTemplateSet), then you could use it here... Then we would have only one place where we initialize the UserAccount

Reply:
`SyncIndex` value changed in 2486906